src/app/static/src/app/components/header/DataExplorationSupportMenu.vue

```diff
@@ -10,31 +11,79 @@
                          v-translate="'axe'">
             </router-link>
         </drop-down>
+        <error-report :open="errorReportOpen"
+                      @send="sendErrorReport"
+                      @close="toggleErrorReportModal">
+            <template v-slot:sectionView>
+                <div>
+                    <label for="section" v-translate="'section'"></label>
+                    <select class="form-control" v-model="section" id="section">
+                        <option v-translate="'dataExploration'"
+                                value="dataExploration">
+                        </option>
+                    </select>
+                </div>
+            </template>
+        </error-report>
     </div>
 </template>
 <script lang="ts">
     import Vue from "vue";
     import DropDown from "./DropDown.vue";
     import i18next from "i18next";
-    import {mapStateProp} from "../../utils";
+    import {mapActionByName, mapStateProp} from "../../utils";
+    import ErrorReport from "../ErrorReport.vue"
     import {Language} from "../../store/translations/locales";
     import {DataExplorationState} from "../../store/dataExploration/dataExploration";
+    import {ErrorReportManualDetails} from "../../types";
 
     interface Computed {
         support: string
         currentLanguage: Language
     }
 
-    export default Vue.extend<unknown, unknown, Computed, unknown>({
+    interface Data {
+        errorReportOpen: boolean
+        section: string
+    }
+
+    interface Methods {
+        toggleErrorReportModal: () => void
+        sendErrorReport: (errorReport: ErrorReportManualDetails) => void
+        generateErrorReport: (payload: ErrorReportManualDetails) => void
+    }
+
+    export default Vue.extend<Data, Methods, Computed, unknown>({
+        data: function () {
+            return {
+                errorReportOpen: false,
+                section: ""
+            }
+        },
         computed: {
             currentLanguage: mapStateProp<DataExplorationState, Language>(null,
                 (state: DataExplorationState) => state.language),
             support() {
                 return i18next.t("support", this.currentLanguage)
             }
         },
+        methods: {
+            generateErrorReport: mapActionByName(null, "generateErrorReport"),
+            toggleErrorReportModal() {
+                this.errorReportOpen = !this.errorReportOpen
+                this.errorReportOpen ? this.section = "dataExploration" : this.section = "";
+            },
+            async sendErrorReport(errorReport) {
+                await this.generateErrorReport({
+                    section: this.section,
```

Comment:
I think this can be simplified a little - no need to use a data property to track "section", since it's value will always be "dataExploration". So here
```suggestion
                    section: i18next.t("dataExploration", this.currentLanguage)
```

Reply:
then can delete lines 74 and 81 and simplify the `Data` interface to just 
```
 interface Data {
        errorReportOpen: boolean
        section: string
    }
```